<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionMasa</class>
 <widget class="QDialog" name="ConversionMasa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CONVERSIONES MASA</string>
  </property>
  <widget class="QLabel" name="lb_mostrar_masas">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>190</y>
     <width>81</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rB_g">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Masa en g</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rB_kg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Masa en kg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rB_oz">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Masa en oz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>561</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Objetos</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_m_obj1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_m_obj2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_m_obj3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_m_obj4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_m_obj5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_bascula">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cB_conversiones_masa">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1.5px solid  rgb(0, 170, 255);
font-weight: bold;</string>
   </property>
   <item>
    <property name="text">
     <string>g a kg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>g a oz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kg  a g</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kg a oz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>oz a g</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>oz a kg</string>
    </property>
   </item>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_convertir_masa">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(0, 211, 211);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Convertir</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_unidades_masa">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);
text-align: right;

</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_volver">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(255, 0, 0);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string> VOLVER</string>
   </property>
   <property name="icon">
    <iconset resource="imagen.qrc">
     <normaloff>:/new/prefix1/recursos/icono_volver.png</normaloff>:/new/prefix1/recursos/icono_volver.png</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convertir de:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cambiar unidades de medida&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;&amp;quot;Seleccione un objeto para medir su masa en la balanza&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>lb_bascula</zorder>
  <zorder>cB_conversiones_masa</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>btn_convertir_masa</zorder>
  <zorder>lb_unidades_masa</zorder>
  <zorder>lb_mostrar_masas</zorder>
  <zorder>btn_volver</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections/>
</ui>
